terragrunt/modules/release-distribution/lambdas/static-router/index.js

```diff
@@ -35,5 +35,12 @@ echo "    curl https://sh.rustup.rs -sSf | sh"
         return redirect('https://sh.rustup.rs', body, callback);
     }
 
+    // Links to `/doc/master/*` are probably from old external sources, so send
+    // them back through the doc site to take advantage of any helpful error
+    // handling that may be defined there
+    if (request.uri.startsWith('/doc/master')) {
+        return redirect('//doc.rust-lang.org/stable' + request.uri.slice(11), callback);
```

Comment:
```suggestion
        return redirect('https://doc.rust-lang.org/stable' + request.uri.slice(11), callback);
```
I think it's better if we force https 👍

Reply:
done in 5b84e4d